spec/system/provider_interface/reject_an_application_spec.rb

```diff
@@ -5,34 +5,31 @@
   include ProviderUserPermissionsHelper
   include CourseOptionHelpers
 
-  scenario 'giving reasons for rejection', with_audited: true do
-    FeatureFlag.deactivate(:structured_reasons_for_rejection_redesign)
-
+  scenario 'giving rejection reasons using redesigned rejection form', with_audited: true do
```

Comment:
for those two system specs just to double check we moved the new R4R from the other specs deleted into the old ones?

Reply:
Yes that's right, `rejection_reasons_spec -> reject_an_application_spec` and the feedback one too.

Reply:
Perhaps I should remove all references to `redesign` now as it's not relevant?

Reply:
we can do that in the next cleanup PR might be easier, don't mind either way